giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/core/files/impl/DocumentDatabaseClient.java

```diff
@@ -88,6 +81,6 @@ protected IDocument getById(String id) {
 
     @Override
     protected EntityManager getClient() {
```

Comment:
is this method still needed?

Reply:
Since we need to extend DatabaseClient for generating id's we cant remove the method until we remove it from DatabaseClient and I noticed Nepomuk and September use the method. Hence I'll be returning null in the method.

Reply:
we can probably remove the dependency to `DatabaseClient`. Pretty much all methods in there use `getClient`, so we can't use any of them anymore (except the one generating an id, which we can move here or somewhere else).

Reply:
same is true for probably all the other classes dependent on `DatabaseClient`